src/codegen/tests/snapshots/rusty__codegen__tests__code_gen_tests__XXXXX.snap.new

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Did you mean to commit this snap? Either it's name is wrong or you missed an insta-review

Reply:
accidentally slipped in from a different branch, I deleted it